### DISCUSSION TITLE
docs: "API Reference" links for components are broken

### DISCUSSION THREAD

**Discussion by QWu4xYV** [cschroeter/park-ui#330] [General]
For example, the "API Reference" Link for the Menu component links to 

https://ark-ui.com/components/docs/menu#api-reference

, but the actual docs are at 

https://ark-ui.com/[framework]/docs/components/menu

Obviously non-blocking, but just as an FYI :)

**Comment by marviobezerra** [comment]
On the same note, the documentation show the following import which is not correct based on `@ark-ui/react@3.0.2`

#### Incorrect
```
import { RadioGroup } from '@ark-ui/react/radio-group';
```

#### Correct
```
import { RadioGroup } from '@ark-ui/react';
```
